build.yaml

```diff
@@ -506,6 +506,41 @@ steps:
    publishAs: batch
    dependsOn:
      - service_base_image
+ - kind: runImage
+   name: render_grafana_nginx_conf
+   image:
+     valueFrom: service_base_image.image
+   script: |
+     set -ex
+     cd /io
+     rm -rf repo
+     mkdir repo
+     cd repo
+     {{ code.checkout_script }}
+     cd grafana
+     if [[ {{ deploy }} = True ]]; then
+      DEPLOY=true
+     else
+      DEPLOY=false
+     fi
+     python3 ../ci/jinja2_render.py '{"deploy": '${DEPLOY}', "default_ns": {"name": "{{ default_ns.name }}"}}' nginx.conf nginx.conf.out
```

Comment:
I think the Jinja-way would be:
```
{% if deploy %}
DEPLOY=true
{% else %}
DEPLOY=false
{% endif %}
```

templating combined with bash conditions gives me the heebie-jeebies.